### PR TITLE
Create ingest-from-coreflux-broker.md

### PR DESCRIPTION
**Pull request by fahadullahshah261**
Hi Docs team! I created this PR about the integration guide for data ingestion from Coreflux into RisingWave. Please review it and let me know your feedback and suggestions. Thanks!

### CODE REVIEW

**Review by hengm3467 — APPROVED** [review]
I did some initial updates and also applied the updates to 2.0. Further improvements might be needed, but as we are in a bit of rush, it would be good to merge this PR tomorrow (Sept 27).

File: docs/ingest/ingest-from-coreflux-broker.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+You will receive an email confirming the creation of your Coreflux broker, including the broker address and login credentials.
+
+![Coreflux Broker Creation: Address and Login Credentials](https://github.com/user-attachments/assets/a9865d4e-392e-49ca-8412-9864b2fc0304)
+
+You can use [MQTT Explorer](http://mqtt-explorer.com/), an intuitive tool for connecting to your MQTT broker. To manage users, such as adding, removing, or changing passwords via MQTT commands, refer to the [Coreflux guide](https://docs.coreflux.org/getting-started/mqtt-explorer-and-coreflux-cloud-broker/).
+
+You are now ready to connect to your Coreflux Cloud broker using various clients. These clients allow you to both publish and subscribe to messages on your MQTT topics. For illustration, we'll use IoT data from a manufacturing scenario, such as monitoring machine performance and detecting faults. Using the Python Paho MQTT client, we will publish this data to the MQTT topic `factory/machine_data`.
+
+---
```
Comment (WanYixian) [inline docs/ingest/ingest-from-coreflux-broker.md]:
```suggestion

```

File: versioned_docs/version-2.0/ingest/ingest-from-coreflux-broker.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+You will receive an email confirming the creation of your Coreflux broker, including the broker address and login credentials.
+
+![Coreflux Broker Creation: Address and Login Credentials](https://github.com/user-attachments/assets/a9865d4e-392e-49ca-8412-9864b2fc0304)
+
+You can use [MQTT Explorer](http://mqtt-explorer.com/), an intuitive tool for connecting to your MQTT broker. To manage users, such as adding, removing, or changing passwords via MQTT commands, refer to the [Coreflux guide](https://docs.coreflux.org/getting-started/mqtt-explorer-and-coreflux-cloud-broker/).
+
+You are now ready to connect to your Coreflux Cloud broker using various clients. These clients allow you to both publish and subscribe to messages on your MQTT topics. For illustration, we'll use IoT data from a manufacturing scenario, such as monitoring machine performance and detecting faults. Using the Python Paho MQTT client, we will publish this data to the MQTT topic `factory/machine_data`.
+
+---
```
Comment (WanYixian) [inline versioned_docs/version-2.0/ingest/ingest-from-coreflux-broker.md]:
```suggestion

```

**Review by WanYixian — APPROVED** [review]
LGTM and removed the horizontal line as headings already split doc into 2 main parts

**Pull request merged**